mitiq/pec/decomposition/tests/test_depolarizing.py

```diff
@@ -0,0 +1,95 @@
+# Copyright (C) 2020 Unitary Fund
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+from typing import List, Tuple
+
+import numpy as np
+import pytest
+from cirq import CCNOT, CNOT, CZ, ISWAP, LineQubit, Operation, SWAP
+
+from mitiq.pec.decomposition.depolarizing import (
+    depolarizing_decomposition,
+    NON_ID_PAULIS,
+)
+
+
+def decomposition_overhead(
+    decomposition: List[Tuple[float, List[Operation]]]
+) -> float:
+    """The overhead of a particular decomposition is the sum of the absolute
+    values of the coefficients of the quasi-probability representation (QPR).
+    """
+    return float(np.sum(np.abs([a for a, _ in decomposition])))
+
+
+def single_qubit_depolarizing_optimal_overhead(noise_level: float) -> float:
+    """See [Temme2017]_ for more information.
+
+    .. [Temme2017] : Kristan Temme, Sergey Bravyi, Jay M. Gambetta,
+        "Error mitigation for short-depth quantum circuits,"
+        *Phys. Rev. Lett.* **119**, 180509 (2017),
+        (https://arxiv.org/abs/1612.02058).
+    """
+    epsilon = 4 / 3 * noise_level
+    return (1 + epsilon / 2) / (1 - epsilon)
+
+
+def two_qubit_depolarizing_optimal_overhead(noise_level: float) -> float:
+    """See [Temme2017]_ for more information.
+
+        .. [Temme2017] : Kristan Temme, Sergey Bravyi, Jay M. Gambetta,
+            "Error mitigation for short-depth quantum circuits,"
+            *Phys. Rev. Lett.* **119**, 180509 (2017),
+            (https://arxiv.org/abs/1612.02058).
+    """
```

Comment:
I think the optimality was shown in:
```
     .. [Takagi2020] : Ryuji Takagi,
        "Optimal resource cost for error mitigation,"
        (https://arxiv.org/abs/2006.12509).
```
Otherwise we could keep reference to `[Temme2017]_` but remove the word `optimal`. In the tests we are actually not interested in the overhead being optimal.

Reply:
Sounds good -- I'll remove optimal